code/modules/clothing/under/miscellaneous_vr.dm

```diff
@@ -596,3 +596,23 @@
 	icon = 'icons/inventory/uniform/item_vr.dmi'
 	default_worn_icon = 'icons/inventory/uniform/mob_vr.dmi'
 	icon_state = "striped_clown_uniform"
+
+/obj/item/clothing/under/dress/cdress_fire
+	name = "flame dress"
+	desc = "A small black dress with a flames print on it. Perfect for recoloring!"
+	icon = 'icons/inventory/uniform/item_vr.dmi'
+	default_worn_icon = 'icons/inventory/uniform/mob_vr.dmi'
+	icon_state = "cdress_fire"
+
+/obj/item/clothing/under/dress/cbridesmaid
+	name = "fancy dress, recolorable"
```

Comment:
I suggest not putting 'recolorable' or 'colorable' on actual item names, only loadout names